debian/copyright

```diff
@@ -0,0 +1,43 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ismrmrd
+Upstream-Contact: <preferred name and address to reach the upstream project>
+Source: <url://example.com>
```

Comment:
Some boiler plate in this file that would need to be replaced, e.g. license, etc.